apps/site/assets/ts/schedule/components/schedule-finder/TripDetails.tsx

```diff
@@ -66,9 +67,18 @@ export const TripDetails = ({
 
   if (!tripInfo) return null;
 
+  const crowding = tripInfo.vehicle ? tripInfo.vehicle.crowding : null;
+
   return (
     <table className="trip-details-table">
       <thead>
+        {crowding && (
+          <tr>
+            <th>
+              <CrowdingPill crowding={crowding} />
```

Comment:
Could you please add a quick test for rendering this (i.e. mounting `TripDetails` for a trip with crowding data and checking for the pill)?